zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -450,6 +459,21 @@ protected InterpreterResult jobRun() throws Throwable {
         p.settings.setParams(settings.getParams());
       }
 
+      // After the paragraph is executed,
+      // need to apply the paragraph to the configuration in the
+      // `interpreter-setting.json` config
+      InterpreterSettingManager intpSettingManager
+          = this.note.getInterpreterSettingManager();
+      if (null != intpSettingManager) {
+        InterpreterGroup intpGroup = interpreter.getInterpreterGroup();
+        if (intpGroup instanceof ManagedInterpreterGroup) {
+          String name = ((ManagedInterpreterGroup) intpGroup).getInterpreterSetting().getName();
+          Map<String, Object> config
+              = intpSettingManager.getConfigSetting(name);
+          applyConfigSetting(config);
+        }
+      }
+
```

Comment:
What if user change the config ? e.g. User change the `displayTitle` to false in frontend for submarine interpreter, then after user execute the paragraph, the config will be reset ? Will `displayTitle` become true again ?